.github/workflows/ci.yml

```diff
@@ -44,13 +44,13 @@ jobs:
             autoconf \
             bison \
             flex \
```

Comment:
Unrelated to this: I wondered before why bison and flex is installed, should not be needed? guess it was inherited from the old CI config somehow?

Reply:
Considering we pass `YACC` explicitly, originally there was an intention to generate the code, but somehow maintainer-mode is disabled thus they are unused. But I think it's better to re-generating the code.

Reply:
Hmm, but I think it's better to use the source code included in repo for release artifacts.

Reply:
Intension was to always generate and not have flex/yacc output in the repo at all? but maybe it makes sense to run flex/yacc as a check/lint for CI?
For release i agree we should not run flex/yacc